runtime.md

```diff
@@ -1,5 +1,51 @@
 # Runtime and Lifecycle
 
+## State
+
+The runtime state for a container is persisted on disk so that external tools can consume and act on this information.
+The runtime state is stored in a JSON encoded file.
+It is recommended that this file is stored in a temporary filesystem so that it can be removed on a system reboot.
+On Linux based systems the state information is stored in `/run/oci`.
+The directory structure for a container is `<root>/<containerID>/state.json`.
+
+* **id** (string) ID is the container's ID.
+* **pid** (int) Pid is the ID of the main process within the container.
+* **root** (string) Root is the path to the container's root filesystem specified in the configuration.
```

Comment:
On Wed, Aug 26, 2015 at 10:43:24AM -0700, Brandon Philips wrote:
> > +* **id** (string) ID is the container's ID.
> > +* **pid** (int) Pid is the ID of the main process within the container.
> > +* **root** (string) Root is the path to the container's root filesystem specified in the configuration.
> 
> can we document the handful of use cases: 

I put some notes in the meeting's Google docs [1], in case folks want
reminders when writing this up for an informative section of the spec.

[1]: https://docs.google.com/document/d/1niKxf4-JWZxHh45bnke4Dl8xoR0QS1ctJ8kVPqS-OKQ/edit?usp=sharing